ext/spl/tests/dit_002.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 SPL: DirectoryIterator defaults
---SKIPIF--
-<?php if (!extension_loaded("spl") || !extension_loaded('reflection') || !defined('GLOB_ERR')) print "skip"; ?>
```

Comment:
`GLOB_ERR` is conditionally defined in https://github.com/php/php-src/blob/a7fe2570d3ce6915d4ea85c62c0f880ddc225ba7/ext/standard/dir.c#L184, so probably that check should remain.